hooks/tk-maya/render_media.py

```diff
@@ -48,21 +49,30 @@ def render(
         # For more informations about the playblast API,
         # https://help.autodesk.com/view/MAYAUL/2020/ENU/?guid=__CommandsPython_playblast_html
 
+        is_windows = sys.platform == "win32"
+
         playblast_args = {
             "viewer": False,  # Specify whether a viewer should be launched for the playblast.
             "forceOverwrite": True,  # Overwrite existing playblast files which may have the the same name as the one specified with the "-f" flag
-            "format": "qt",  # The format of the output of this playblast.
             "percent": 100,  # Percentage of current view size to use during blasting.
+            # The format of the output of this playblast.
         }
 
+        # The format of the output of this playblast.
+        # Rendering a .mov does not work out of the box on Windows.
+        # It requires to have QuickTime installed and this is not really supported...
+        # Since Shotgun Create also supports avi files, it's a safe move to render this type on Widows
+        playblast_args["format"] = "avi" if is_windows else "qt"
```

Comment:
How about Linux? Quicktime on Linux? Might be worth a comment because to the uninitiated eye it looks like a bug.

Reply:
I get told by a Maya engineer that Maya use an open source library for the qt format.

Reply:
Alright. Still, please leave a comment in saying that Maya supports Qt out of the box on Linux.